### PR TITLE
[4650][next] Add notes in Publishing screens

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/4650

### CODE REVIEW

File: ui/app/src/utils/i18n-legacy.ts [hunk 1]
```diff
@@ -681,6 +681,15 @@ export const publishingMessages = defineMessages({
   queued: {
     id: 'publishing.queued',
     defaultMessage: 'Queued'
+  },
+  bulkPublishNote: {
+    id: 'publishing.bulkPublishNote',
+    defaultMessage: 'Bulk publish should be used to publish changes made in Studio via the UI.'
```
Comment (rart) [inline ui/app/src/utils/i18n-legacy.ts]:
```suggestion
    defaultMessage: 'Bulk publish should be used to publish changes made in Studio via the UI. For changes made via direct git actions, use the "publish by" feature.'
```

File: ui/app/src/utils/i18n-legacy.ts [hunk 2]
```diff
@@ -681,6 +681,15 @@ export const publishingMessages = defineMessages({
...
+  },
+  publishByNote: {
+    id: 'publishing.publishByNote',
+    defaultMessage:
+      'Publish by commit ID(s) must be used for changes made via direct git actions against the repository or pulled from a remote repository.'
```
Comment (rart) [inline ui/app/src/utils/i18n-legacy.ts]:
```suggestion
      'Publish by commit ID(s) must be used for changes made via direct git actions against the repository or pulled from a remote repository. For changes made via Studio on the UI, use "bulk publish".'
```

File: static-assets/ng-views/publishing.html [hunk 3]
```diff
@@ -50,6 +50,9 @@ <h2 class="page-header">{{ 'admin.publishing.PUBLISHING' | translate }}</h2>
       </tab-heading>
       <div class="m10 mt25 bulkContainer">
         <h3 class="mt5 mb15">{{ 'admin.publishing.BULKPUBLISH' | translate }}</h3>
+        <p class="section-note">
+          {{ publish.bulkPublishNote }}
```
Comment (rart) [inline static-assets/ng-views/publishing.html]: Please add a warning icon

File: static-assets/ng-views/publishing.html [hunk 4]
```diff
@@ -144,6 +147,9 @@ <h3 class="mt5 mb15">{{ 'admin.publishing.BULKPUBLISH' | translate }}</h3>
       </tab-heading>
       <div class="m10 mt25 commitsContainer">
         <h3 class="mt5 mb15">{{ 'admin.publishing.PUBLISHBYCOMMITS' | translate }}</h3>
+        <p class="section-note">
+          {{ publish.publishByNote }}
```
Comment (rart) [inline static-assets/ng-views/publishing.html]: Please add a warning icon

**Review by rart — CHANGES_REQUESTED** [review]
Please apply changes to 3.1.x PR

**Review by rart — APPROVED** [review]

File: ui/app/src/utils/i18n-legacy.ts [hunk 5]
```diff
@@ -684,12 +684,13 @@ export const publishingMessages = defineMessages({
...
     id: 'publishing.bulkPublishNote',
-    defaultMessage: 'Bulk publish should be used to publish changes made in Studio via the UI.'
+    defaultMessage:
+      'Bulk publish should be used to publish changes made in Studio via the UI. For changes made via direct git actions, use the "Publish by..." feature.'
   },
   publishByNote: {
     id: 'publishing.publishByNote',
     defaultMessage:
-      'Publish by commit ID(s) must be used for changes made via direct git actions against the repository or pulled from a remote repository.'
+      '"Publish by..." feature must be used for changes made via direct git actions against the repository or pulled from a remote repository. For changes made via Studio on the UI, use "Bulk Publish".'
```
Comment (sumerjabri) [inline ui/app/src/utils/i18n-legacy.ts:693]: Is the `...` intentional here?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: ui/app/src/utils/i18n-legacy.ts (hunk 5 above)
Comment (jvega190) [reply ui/app/src/utils/i18n-legacy.ts:693]: @sumerjabri yes it is, that's the label of the tab in 4.0.x

Comment (rart) [reply ui/app/src/utils/i18n-legacy.ts:693]: @sumerjabri publish by commit was relabelled to "publish by..." as a consequence of the ticket that requested publishing tags.

image

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**